oligo_designer_toolsuite/oligo_specificity_filter/_filter_seed_region.py

```diff
@@ -0,0 +1,196 @@
+import os
+import re
+from pathlib import Path
+from subprocess import Popen
+
+import pandas as pd
+from joblib import Parallel, delayed
+
+from . import SpecificityFilterBase
+
+
+class BowtieSeedRegion(SpecificityFilterBase):
+    """This class filters probes based on the Bowtie short read alignment tool on a specific region of the probe. The region take in consideration is created according to the ``seed_region_creation`` class.
+    The user can customize the filtering by specifying the num_mismatches, and all probes with number mismatches lower or equal to min_mismatches inside the mismatch_region are filtered out.
+
+    Use conda install -c bioconda bowtie to install Bowtie package
+
+    :param file_transcriptome_fasta: path to fasta file containing all probes
+    :type file_transcriptome_fasta: str
+    :param seed_region_creation: The region of the probe where the mismatches are considered. Probes that have less than min_mismatches in the first L bases (where L is 5 or greater) are filtered out
```

Comment:
"In the first L" bases might be confusing since it can also be in the middle of the probe. Maybe rephrase

Reply:
Did you address this comment?